governance-exercises/Interpretation Exercise Response.md

```diff
@@ -0,0 +1,46 @@
+**Do you think there is any ambiguity in these clauses? If so, where, and how would you rewrite the clauses to remove the ambiguity?**
+
+Yes, recent events over the EDSR made it very clear ambiguity exists. Various community members were unsure on if this clause would allow for such a vote. One reasonable interpretation is that this power was meant to be used to fix obvious errors or weird edge cases that break things, but the author clarified that it could be used to deal with broader protocol issues(such as low DSR utilization and declining DAI supply). 
+
+Given the author's intent, I would suggest  the following edits be included in a future Scopes update to clarify that the power is intended to be this broad. To be clear, this does not necessarily mean I believe Governance Facilitators ought to have this power, but that this is the intent of of the author and majority of voting MKR.
+
+**12.1: Bootstrapping Governance Votes**
+
+Governance Facilitators are empowered to use broad judgment when exercising their ability to make direct Atlas interpretations and edits to the Atlas and the Scope Artifacts, when this relates to unintended consequences or mistakes in the affected documents, *as well as when this relates to time-sensitive issues with the Launch Project.*
+
+12.1.1: In case of unintended consequences or mistakes in the Atlas and Scope Artifacts causing Maker Governance to function incorrectly, the Governance Facilitators can at any time run a Governance Poll to enable MKR holders to make a decision that will alleviate and resolve unintended consequences or mistakes.
+
+12.1.2: The Governance Facilitators can at any time propose to edit any content of a Scope Artifact through a Governance Poll.
```

Comment:
The core ambiguity here remains unresolved. Namely that:
12.1 + 12.1.1 says: This power can be used in these circumstances.
and 12.1.2 says: This power can be used at any time.